src/test/resources/nuts_synthetic_one/test-nuts-four.geojson

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Can you give a meaningful name? like **synthetic_nuts_with_4_coordinates.geojson** ?